css/index.css

```diff
@@ -1 +1,306 @@
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+}
+
+.nav-link {
+    width: 100%;
+    text-decoration: none;
+    border-top: 0.1rem solid var(--lightAccent);
+    text-align: center;
+    color: var(--darkAccent);
+    padding: 1.5rem;
+}
+
+.nav-link:last-child {
+    border-bottom: 0.1rem solid var(--lightAccent);
+}
+
+.nav-link:hover {
+    color: var(--lightAccent);
+    background-color: var(--dark);
+}
+
+/* Header
+============================ */
+
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    padding: 3rem;
+}
+
+.big-text {
+    font-size: 7rem;
+    font-family: var(--secondaryFont);
+}
+
+.header-img {
+    display: none;
+}
+
+.button {
+    max-width: 25rem;
+    color: var(--dark);
+    background-color: var(--light);
+    border: 0.1rem solid var(--dark);
+    padding: 1rem 6rem;
+    margin: 3rem 0;
+}
+
+.button:hover {
+    color: var(--light);
+    background-color: var(--dark);
+}
+
+/* Top Section
+============================ */
+
+.top-section  * {
+    padding: 1rem;
+}
+
+.title {
+    font-family: var(--secondaryFont);
+}
+
+/* Middle Section
+============================ */
+
+.middle-img {
+    width: 100%;
+    padding: 0 1%;
+}
+
+/* Bottom Section
+============================ */
+
+.bottom-section  * {
+    padding: 1rem;
+}
+
+/* Footer Section
+============================ */
+
+.contact {
+    padding: 2rem;
+    max-width: 25rem;
+}
+
+.address {
+    padding: 1rem 0;
+}
+
+.contact-link {
+    text-decoration: none;
+    line-height: 2;
+    color: var(--dark);
+}
+
+.contact-link:hover {
+    color: var(--lightAccent);
+}
+
+.copyright {
+    text-align: center;
+    padding: 2rem;
+}
+
+
+/* Tablet View
+============================ */
+
+@media (min-width: 500px) {
+
+    /* Navigation */
+
+    nav {
+        flex-direction: row;
+        justify-content: space-evenly;
+    }
+
+    .nav-link {
+        border: none;
+    }
+
+    .nav-link:last-child {
+        border: none;
+    }
+
+    .nav-link:hover {
+    color: var(--dark);
+    background-color: var(--light);
+    transform: scale(1.2);
+    transition: 300ms;
+    }
+
+    /* Top Section */
+
+    .top-section {
+        display: flex;
+    }
+
+    /* Bottom Section */
+
+    .bottom-section {
+        display: flex;
+    }
+}
+
+/* Desktop View
+============================ */
+
+@media (min-width: 800px) {
+
+    /* Container */
+
+    .container {
+        max-width: 880px;
+        margin: 0 auto;
+    }
+
+    /* Navigation Container */
+
+    .nav-container {
+        display: flex;
+        align-items: baseline;
+        flex-direction: row-reverse;
+    }
+
+    /* Logo */
+
+    .logo {
+        padding: 0;
+    }
+
+    /* Header */
+
+    .header {
+        flex-direction: row;
+        justify-content: space-between;
+    }
+
+    .header-img {
+        display: block;
+        width: 100%;
```

Comment:
Is there a reason I'm not seeing that your max-width should be smaller than your width here?